### DISCUSSION TITLE
Potential time to rebrand?

### DISCUSSION THREAD

**Discussion by splashsky** [Annoraaq/grid-engine#67] [General]
The scope of your wonderful plugin has well passed mere "grid movement", my friend. It might be time to refactor the repo and the plugin to something like `Annogrid`, or `GridEngine`, no? It'll be a more marketable name and a lot shorter to work with than `phaser-grid-movement-plugin` and `gridMovementPlugin`, y'know?

Just an idea. 😉

**Comment by Annoraaq** [comment]
Good idea. I like GridEngine. I will create an issue because it will involve renaming a lot of links and deprecate the npm package.
